### DISCUSSION TITLE
Write an "about" document titled "Distributing a Plaster Template"

### DISCUSSION THREAD

**Discussion by daviwil** [PowerShellOrg/Plaster#375] [Show and tell]
In addition to "Creating a Plaster Manifest", we'll also need an about document describing how to distribute a Plaster template via the PowerShell Gallery.

**Comment by larssb** [comment]
I created a project for containing Plaster templates named [PlasterPlethora](https://github.com/larssb/PlasterPlethora). I thought I might as well kick-start the community with this. Hopefully it can be a starter for getting us going with a way to share Plaster templates. I hope people will find it useful. I'm all ears for comments and feedback.

Kind regards.
